docs-rtd/howto/manage-clouds.md

```diff
@@ -33,15 +33,16 @@ resource "juju_model" "my-model" {
 At present the Terraform Provider for Juju supports cloud access management only for clouds added to a Juju controller added to JIMM.
 ```
 
-When using Juju with JAAS, to grant one or more users, groups, and/or service accounts access to a JAAS-known cloud, in your Terraform plan add a resource type `juju_jaas_access_cloud`, specifying the cloud name, the JAAS cloud access level, and the desired list of users, groups, and/or service accounts. For example:
+When using Juju with JAAS, to grant one or more users, service accounts, roles, and/or groups access to a JAAS-known cloud, in your Terraform plan add a resource type `juju_jaas_access_cloud`, specifying the cloud name, the JAAS cloud access level, and the desired list of users, service accounts, roles, and/or groups. For example:
```

Comment:
Thought: This is a very long sentence which is hard to piece apart. Can it be broken up? Same across the other .md files with a similar sentence. And others.